### PR TITLE
Update kotlin compiler version to 1.3.61

### LINKED ISSUE #531: Update bundled compiler version to 1.3.61
To support klib it's necessary to update dokka bundled compiler version to 1.3.60

### PR DESCRIPTION
**Pull request by kamilok1965**
Fixes #531

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```
Comment (martinbonnin) [inline .idea/codeStyles/codeStyleConfig.xml]: Is that intentional ?

File: .idea/compiler.xml [hunk 2]
```diff
@@ -11,14 +11,6 @@
...
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <annotationProcessing>
-      <profile name="Maven default annotation processors profile" enabled="true">
-        <sourceOutputDir name="target/generated-sources/annotations" />
-        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
-        <outputRelativeToContentRoot value="true" />
-        <module name="dokka-maven-plugin" />
-      </profile>
-    </annotationProcessing>
```
Comment (martinbonnin) [inline .idea/compiler.xml]: Should this file be removed from source control ?

Comment (kamilok1965) [reply .idea/compiler.xml]: Probably, but the decision was to keep the entire `.idea` directory to help setup this project, so we sometimes just update it

File: .idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (kamilok1965) [reply .idea/codeStyles/codeStyleConfig.xml]: Not sure TBH. Intellij changed it, doesn't look harmful

File: .idea/compiler.xml (hunk 2 above)
Comment (martinbonnin) [reply .idea/compiler.xml]: I usually only commit the codestyle settings (`.idea/codeStyles/`). The problem of adding other files is that it creates noise in the diffs.

**Pull request merged**